plugins/logs/plugin.go

```diff
@@ -620,6 +638,22 @@ func (p *Plugin) Log(ctx context.Context, decision *server.Info) error {
 		inputAST:       decision.InputAST,
 	}
 
+	headers := map[string][]string{}
+	rctx := p.config.RequestContext
+
+	if rctx.HTTPRequest != nil && len(rctx.HTTPRequest.Headers) > 0 && decision.HTTPRequestContext.Header != nil {
+		for _, h := range rctx.HTTPRequest.Headers {
+			values := decision.HTTPRequestContext.Header.Values(h)
+			if len(values) > 0 {
+				headers[h] = decision.HTTPRequestContext.Header.Values(h)
```

Comment:
Nothing in particular. In general though, this is a whole category of XXS attacks that target the consumer of the log. In part, this is a consideration for the consumer-side, but it could be argued that OPA, which is generating the log, should take some of the burden to sanitize the content; especially as some of the values are created wholesale by the requesting client, and not OPA.